package.json

```diff
@@ -54,6 +54,7 @@
     "typescript": "5.1.6"
   },
   "dependencies": {
+    "@changesets/cli": "^2.26.2",
```

Comment:
Should we drop the `semantic-release` dependency in this PR too?

Reply:
Good spot, removed now.